guides/common/modules/proc_registering-capsule-to-satellite-server.adoc

```diff
@@ -1,11 +1,7 @@
 [id="Registering_Proxy_to_Server_{context}"]
 = Registering to {ProjectServer}
 
-Use this procedure to register the base operating system on which you want to install {SmartProxyServer} to {ProjectServer}.
-
-ifdef::katello[]
-Registering your {SmartProxyServer} as a content host is optional unless you wish to download the installation packages from your synced repositories.
-endif::[]
+Registering the base operating system on which you want to install {SmartProxyServer} to {ProjectServer} enables you to download the installation packages from your synchronized repositories.
```

Comment:
Style wise it feels like a long sentence. I'm not sure we need to mention `"base operating system". I'd be tempted to say 
```suggestion
Registering the host for your {SmartProxyServer} to {ProjectServer} enables you to download the installation packages from your synchronized repositories.
```

Reply:
If we use it elsewhere too, then I would also stick to it.